<launch>
  <include file="$(find hrc_discrim_learning)/launch/basic_aruco.launch"/>

  <rosparam param="/perception/objs">
    "blue-marker" : 0
    "red-marker"  : 1
    "green-marker": 2
    "pen"         : 3
    "screwdriver" : 4
    "mug"         : 5
    "block"       : 6
    "yellow-ball" : 7
  </rosparam>

  <node name="env_perception" pkg="hrc_discrim_learning" type="env_perception.py" respawn="false" output="screen">
    <!-- <param name="" value=""/> -->
  </node>

</launch>
